### PR TITLE
Invoice

### PR DESCRIPTION
**Pull request by Omegaeye**

### CODE REVIEW

File: app/views/admin/merchants/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<h1 style="text-align:left"> Admin Merchants </h1>
+
+<%= render "/shared/admin_links" %><br>
```
Comment (jbranham1) [inline app/views/admin/merchants/index.html.erb:3]: 👍 👍

File: config/routes.rb [hunk 2]
```diff
@@ -1,8 +1,12 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  resources :customers, :merchants, :invoice_items, :invoices, :transactions, :items
-
   namespace :admin do
     resources :merchants, :invoices
   end
+
+  resources :merchants do
```
Comment (jbranham1) [inline config/routes.rb:7]: Good call!

**Review by jbranham1 — APPROVED** [review]
All of this added css looks good!

**Review by Gvieve — APPROVED** [review]
Thanks so much for adding this in Khoa. Great looking ahead.

**Pull request merged**